TEAM_STRUCTURES.md

```diff
@@ -169,30 +169,30 @@ The Documentation, Education and Websites Working Group creates the Documentatio
 - **Coordination**: https://github.com/ipfs/in-web-browsers
 - **Captain**: [Marcin Rataj](https://github.com/lidel)
 
-The Integration with Web Browsers Working Group designs and implement Browser integrations, Web Extensions, Service Workers and any other strategy that contributes to IPFS being integrated with the Web today.
+The Integration with Web Browsers Working Group designs and implements browser integrations, web extensions, service workers and any other strategy that contributes to IPFS being integrated with the internet today.
```

Comment:
"web" (the document/app/"object" "web") is the correct term here. Integration with the *internet* (the computer network) is more of a libp2p thing.